src/abi.cpp

```diff
@@ -188,6 +188,8 @@ void eosio::convert(const abi_def& abi, eosio::abi& c) {
         c.action_types[a.name] = a.type;
     for (auto& t : abi.tables)
         c.table_types[t.name] = t.type;
+    for (auto& r : abi.action_results.value)
+        c.action_result_types[r.name] = r.result_type;
```

Comment:
The PR title is "return action_results",  here only action_result_types is copied. Is action_results is the same as action_result_types?

Reply:
Each `action_result` consists of a name and a type. The `action_result_types` is just a map of name->type. This is then referenced when calling get `abieos_get_type_for_action_result`